packages/api/src/controllers/animalController.js

```diff
@@ -142,6 +148,23 @@ const animalController = {
           }
 
           individualAnimalResult.group_ids = groupIds;
+
+          const animalUseRelationships = [];
```

Comment:
No worries if it's a pain to explain, but (if it can be done easily) would you kindly sum up for the lazy why we are able to use `insertGraphWithResponse` for batches and not for animals? 😇 Is it because of how we handled groups?

Reply:
I did not look into it -- it should be possible! Maybe if some stuff was moved to middleware. I chose not to re-write the controller since it is somewhat complex with all the extra creation logic for type, breeds, groups etc. We kind of have four forms in one again add type, add breed, add group, add animal.  This was the path of least resistance.

Reply:
Sorry it's on me... Looking at it now, I feel that there might have been a way to use `insertGraphWithResponse` while handling groups.